source/renderer/app/containers/LoadingPage.js

```diff
@@ -72,17 +74,26 @@ export default class LoadingPage extends Component<InjectedProps> {
           currencyIcon={adaLogo}
           apiIcon={cardanoLogo}
           cardanoNodeState={cardanoNodeState}
+          currentAppVersion={version}
+          availableAppVersion={availableAppVersion}
           isConnected={isConnected}
           isSynced={isSynced}
           isNodeStopping={isNodeStopping}
           isNodeStopped={isNodeStopped}
           isNotEnoughDiskSpace={isNotEnoughDiskSpace}
+          isNewAppVersionAvailable={isNewAppVersionAvailable}
+          isNewAppVersionLoading={getLatestAppVersionRequest.isExecuting}
+          isNewAppVersionLoaded={
```

Comment:
@nikolaglumac can you isolate logic in separated variable outside component parameter declaration.

Reply:
@tomothespian good point! Fixed now 🚀